datasets/thelook_ecommerce/pipelines/thelook_ecommerce/pipeline.yaml

```diff
@@ -79,7 +79,7 @@ dag:
 
         # Set the environment variables you need initialized in the container. Use these as input variables for the script your container is expected to perform.
         env_vars:
-          NUM_OF_USERS: "15000"
+          NUM_OF_USERS: "100000"
```

Comment:
As mentioned above, we can set a new env var called `NUM_GHOST_EVENTS` (or similar)